### PR TITLE
add slight personalization to portfolio

### PR DESCRIPTION
**Pull request by utibeno**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,18 +1,90 @@
...
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+    <section class="hero">
+      <img src="images/portfolio_pic.jpg" alt="Uty" />
```
Comment (classm-sps) [inline portfolio/src/main/webapp/index.html]: I would expect to see this image being added to your repo in this commit. Without that, it won't be possible to fully publish your site.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,18 +1,90 @@
...
+      <div class="paragraph">
+        <p>
+          Hey guys, my name is Utibeno Ekpoudom, but Uty is fine as well. I am
+          currently an undergrad studying Computer Science at Georgia Tech.
+          While my concentrations are in intelligence and Infonetworks, I do
+          appreciate amazing design, especially when it comes to web
+          development. Therefore, In addition to my core concentrations I do
+          enjoy designing visually stunning website UIs. Apart from that, I do
+          enjoy watching TV Show, Altered Carbon, GoT, The Expanse etc. As you
+          can tell by my examples I love SciFi, be it shows or movies.
```
Comment (classm-sps) [inline portfolio/src/main/webapp/index.html]: We like many of the same shows! I'm really excited for season 2 of Altered Carbon popping this week!

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -1,18 +1,90 @@
...
+        </p>
+      </div>
+      <div class="gallery">
+        <img src="images/gatech.jpg" alt="gatech" />
+        <img src="images/nigerianFlag.jpg" alt="" />
+        <img src="images/bugattiDivo.jpg" alt="" />
+        <img src="images/gameOfThrones.jpg" alt="" />
+        <img src="images/tonyStark.jpg" alt="" />
+        <img src="images/alteredCarbon.jpg" alt="" />
```
Comment (classm-sps) [inline portfolio/src/main/webapp/index.html]: All these images need to be added to the repo. Looks like there are some more below, as well

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -1,18 +1,90 @@
...
+      <a href="https://github.com/utibeno"
+        ><img src="https://img.icons8.com/bubbles/50/000000/github.png"
+      /></a>
+      <a href="https://www.linkedin.com/in/utibenoekpoudom/"
+        ><img src="https://img.icons8.com/bubbles/50/000000/linkedin.png"
+      /></a>
+    </footer>
   </body>
-</html>
+</html>
```
Comment (classm-sps) [inline portfolio/src/main/webapp/index.html]: nit: this red "circle slash + carriage return" means that the line doesn't end in a trailing newline. It's good practice for all text files to end with a trailing newline, it tends to work better with lots of tools.

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -1,9 +1,138 @@
...
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
-}
-
 #greeting-container {
   margin-top: 20px;
+  font-family: 'Neucha';
+  color: #FEDA15;
+
```
Comment (classm-sps) [inline portfolio/src/main/webapp/style.css]: please remove this blank line before the closing brace. This applies to several other places in the file.

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (utibeno) [reply portfolio/src/main/webapp/index.html]: yup yup, me too!!

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (utibeno) [reply portfolio/src/main/webapp/index.html]: just added the images folder

File: portfolio/src/main/webapp/index.html (hunk 4 above)
Comment (utibeno) [reply portfolio/src/main/webapp/index.html]: Gotcha, so I should just add an extra line to make it 91?

**Pull request merged**